modules/core/src/v2/coinFactory.ts

```diff
@@ -210,3 +228,16 @@ for (const token of [...tokens.bitcoin.algo.tokens, ...tokens.testnet.algo.token
   const tokenConstructor = AlgoToken.createTokenConstructor(token);
   GlobalCoinFactory.registerCoinConstructor(token.type, tokenConstructor);
 }
+
+for (const token of [...tokens.bitcoin.avaxc.tokens]) {
```

Comment:
what is a `bitcoin.avaxc` ? that doesn't seem great

Reply:
I think that it is part of legacy code where bitcoin is naming mainnet. If you see others coins, all them has bitcoin and testnet. Here is the definition. 

https://github.com/BitGo/BitGoJS/blob/e3f3f67784f392f23cead05b62c91a11eee5709a/modules/core/src/v2/types.ts#L1